identity/deprecated.go

```diff
@@ -0,0 +1,54 @@
+package identity
```

Comment:
What is the purpose of moving these functions to a separate file? It makes the
change harder to trace. If the functions are deprecated, add a deprecation
comment (like you did), but leave the functions in place. That way, it's clear
that the only change to the function is the addition of the deprecation comment.

Reply:
Moved back.